### DISCUSSION TITLE
v129 crosh exploits?

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1302] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
are there any?

**Comment by Luigi-Master** [comment]
maybe try powerwashing your Chromebook and do control d and when you sign in and there is a lock when loading to your previous data and Chromebook background, immediately turn off your wifi and try to open crosh as quickly as possible, I don't know, someone got crosh to open but he wont tell anyone, sorry, don't know the secret.

**Reply by schoolexploitkid** [reply]
i mean i booted into crosh but idk what to do with it

**Reply by ghost** [reply, marked as answer]
if you are lucky enough to have kernver 3 and downgrade to any version below v123 then you could do VMC exploit. But other than that there is no crosh exploit for v129.

**Reply by Luigi-Master** [reply]
install Linux with it or disable extensions using commands

**Reply by schoolexploitkid** [reply]
wdym disable extensions using commands? ik i cant enable linux

**Reply by Luigi-Master** [reply]
Not linux through the settings, with a crosh command, if you don't want to do that, just search the web for a command that disables all extensions including the managed ones. I haven't used terminals like these in a long time so I forgot a lot of the commands.

**Comment by akabutnicer** [comment]
other than OD crosh (or whatever the name was), nah.

**Reply by schoolexploitkid** [reply]
what is OD crosh

**Reply by akabutnicer** [reply]
exactly
